src/aioswitcher/api/packets.py

```diff
@@ -17,16 +17,29 @@
 # weekdays sum, start-time timestamp, end-time timestamp
 SCHEDULE_CREATE_DATA_FORMAT = "01{}01{}{}"
 
+PHONE_ID = "0000"
+DEVICE_PASSWORD = "00000000"
 NO_TIMER_REQUESTED = "00000000"
 NON_RECURRING_SCHEDULE = "00"
 # format values are local session id, timestamp
 REQUEST_FORMAT = "{}340001000000000000000000{}00000000000000000000f0fe"
-PAD_74_ZEROS = "0" * 74
+REQUEST_FORMAT2 = "{}390001000000000000000000{}00000000000000000000f0fe"
 
+REQUEST_FORMAT_BREEZE = "{}000001000000000000000000{}00000000000000000000f0fe"
+PAD_74_ZEROS = "0" * 74
 # format value just timestamp (initial session id is "00000000")
 LOGIN_PACKET = "fef052000232a10000000000" + REQUEST_FORMAT[2:] + "1c" + PAD_74_ZEROS
+
+LOGIN2_PACKET = (
+    "fef030000305a60000000000ff0301000000000000000000{}00000000000000000000f0fe{}00"
+)
+
 # format values are local session id, timestamp, device id
 GET_STATE_PACKET = "fef0300002320103" + REQUEST_FORMAT + "{}00"
+
+# format values are local session id, timestamp, device id
+GET_STATE_PACKET2 = "fef0300003050103" + REQUEST_FORMAT2 + "{}00"
+
 # format values are local session id, timestamp, device id, command, timer
 SEND_CONTROL_PACKET = (
     "fef05d0002320102" + REQUEST_FORMAT + "{}" + PAD_74_ZEROS + "0106000{}00{}"
```

Comment:
```suggestion
    "fef05d0002320102" + REQUEST_FORMAT + "{}" + PAD_72_ZEROS + "000106000{}00{}"
```
We need to do the same fix for all the others bellow (can't suggest them on GitHub)